services/storage/prepare.mk

```diff
@@ -0,0 +1,3 @@
+prepare.storage:
+	@echo "Adding self-signed tls certs to trusted store"
+	@./bin/addCert.sh
```

Comment:
I suggest to remove the trust on `clean`

Reply:
The tls cert is deleted on `clean`. And since the `/etc/ssl/certs` contains `symlink` to this file (not a copy of it) trust is lost.

Reply:
We can discuss it in separate issue.

Reply:
https://github.com/nspcc-dev/neofs-dev-env/issues/109#issue-937950824